### DISCUSSION TITLE
Report Bug

### DISCUSSION THREAD

**Discussion by DKA78** [BG-Software-LLC/SuperiorSkyblock2#1159] [Denied Feature Requests] [answered]
Hello, where does this error come from? it appears as soon as I put a message in the chat since I made the command /is bank

![image](https://user-images.githubusercontent.com/91481619/170494419-fc17021c-2d33-457d-8edc-cef2fc58b9be.png)

**Comment by OmerBenGera** [comment, marked as answer]
Update to latest dev build.
Next time, open an issue instead :)
